app/views/omni_nav/_global_nav.html.erb

```diff
@@ -0,0 +1,74 @@
+<div id="cu-global-nav">
+  <nav>
+    <ul class="global-nav-links">
+      <li class="primary-link">
+        <a>About</a>
+        <div  class="global-nav-dropdown">
+          <ul>
+            <li><a class="icon icon-location" href="../../about/maps-directions/index.aspx">Maps and Directions </a></li>
+            <li><a class="icon icon-california" href="../../about/visit-chapman/index.aspx">Visit Chapman</a></li>
+            <li><a class="icon icon-cu-monogram" href="../../discover/index.html">Discover Chapman</a></li>
+            <li><a class="icon icon-office" href="../../about/campus/index.aspx">Our Campus</a></li>
+            <li><a class="icon icon-cu-monogram" href="../../about/facts-history/index.aspx">Facts and History</a></li>
+            <li><a class="icon icon-cu-window" href="../../about/administration/index.aspx">Administration</a></li>
+            <li><a class="icon icon-envelop" href="../../about/contact-us.aspx">Contact Us</a></li>
```

Comment:
Relative paths always worry me. I'd prefer we use an absolute path consistent with end-user perspective. That is, why wouldn't `href` here just be `/about/contact-us.aspx`?

I know this is inactive dummy code, but it could inform actual implementation so I'd prefer to see it right.